conf/v6.routes

```diff
@@ -9,7 +9,7 @@ GET        /:nino/brought-forward-losses/tax-year/:taxYearBroughtForwardFrom
 # Loss Claims
 
 POST       /:nino/loss-claims                                                   v6.lossClaims.create.CreateLossClaimController.create(nino: String)
-DELETE     /:nino/loss-claims/:claimId                                          v6.lossClaims.delete.DeleteLossClaimController.delete(nino: String, claimId: String)
+DELETE     /:nino/loss-claims/:claimId/tax-year/:taxYear                        v6.lossClaims.delete.DeleteLossClaimController.delete(nino: String, claimId: String, taxYear: String)
```

Comment:
Same here, please use right name `taxYearClaimedFor` in. both places, thanks. See lines 15 and 16.